ecommerce/extensions/payment/views/fomopay.py

```diff
@@ -0,0 +1,375 @@
+                logger.info(
+                    'Payment Authorization was declined for basket [%d]. The payment response was '
+                    'recorded in entry [%d].',
+                    basket.id,
+                    ppr.id,
+                )
+            except PaymentError:
+                logger.exception(
+                    'FOMO Pay payment failed for basket [%d]. The payment response [%s] was recorded in entry [%d].',
+                    basket.id,
+                    notification.get("result", "Unknown Error"),
+                    ppr.id
+                )
+                raise
+            except:  # pylint: disable=bare-except
+                logger.exception(
+                    'Attempts to handle payment for basket [%d] failed. The payment response [%s] was recorded in'
+                    ' entry [%d].',
+                    basket.id,
+                    notification.get("result", "Unknown Error"),
+                    ppr.id
+                )
+                raise
+
+        return basket
+
+    def _get_basket(self, basket_id):
+        """Return basket object from the basket_id."""
+        if not basket_id:
+            return None
+
+        try:
+            basket_id = int(basket_id)
+            basket = Basket.objects.get(id=basket_id)
+            basket.strategy = strategy.Default()
+            Applicator().apply(basket, basket.owner, self.request)
+            return basket
+        except (ValueError, ObjectDoesNotExist):
+            return None
+
+    def create_order(self, request, basket, billing_address):
+        """Place the order and begin fulfillment pipeline."""
+        order_number = OrderNumberGenerator().order_number(basket)
+        try:
+            shipping_method = NoShippingRequired()
+            shipping_charge = shipping_method.calculate(basket)
+
+            # Note (CCB): This calculation assumes the payment processor has not sent a partial authorization,
+            # thus we use the amounts stored in the database rather than those received from the payment processor.
+            order_total = OrderTotalCalculator().calculate(basket, shipping_charge)
+            user = basket.owner
+
+            return self.handle_order_placement(
+                order_number,
+                user,
+                basket,
+                None,
+                shipping_method,
+                shipping_charge,
+                billing_address,
+                order_total,
+                request=request
+            )
+        except Exception:  # pylint: disable=broad-except
+            self.log_order_placement_exception(order_number, basket.id)
+            raise
+
+    def _get_billing_address(self, fomopay_response):
+        """ Fill in the billing address using the payment notification.
+
+        FOMO Pay doesn't provide a billing address in the payment notification,
+        so as a temporal address we're storing the payment id.
```

Comment:
@felipemontoya as the docstring says there's not a billing address in the notification, I filled this information but I think is better to just remove this field, What do you think?